tests/testthat/_snaps/compat-types-check.md

```diff
@@ -7,17 +7,23 @@
       Error in `checker()`:
       ! `foo` must be `TRUE` or `FALSE`, not `NA`.
     Code
-      err(checker(lgl(), check_bool))
+      err(checker(NULL, check_bool))
     Output
       <error/rlang_error>
       Error in `checker()`:
-      ! `foo` must be `TRUE` or `FALSE`, not an empty logical vector.
+      ! `foo` must be `TRUE` or `FALSE`, not NULL.
```

Comment:
Weird that `NULL` isn't backticked here. Possible issue in `obj_type_friendly()`?

Reply:
Good point.